src/webhook/webhook.service.ts

```diff
@@ -13,16 +13,25 @@ export class WebhookService {
     private readonly prismaService: PrismaService,
   ) {}
 
-  async sendWebhookWithRecipe(recipe: RecipeResponse): Promise<void> {
-    const webhookURL = process.env.WEBHOOK_URL;
+  async createRecipeWebhook(recipe: RecipeResponse): Promise<void> {
+    const webhooks = await this.fetchAllWebhooks();
 
-    try {
-      await lastValueFrom(this.httpService.post(webhookURL, recipe));
-    } catch (error) {
-      throw new Error('There was an error while sending request');
+    const createRecipeWebhook = webhooks.filter((webhook) => {
+      if (webhook.name === process.env.CREATE_RECIPE_WEBHOOK_NAME) {
+        return webhook;
+      }
+    });
+
+    if (createRecipeWebhook[0].isEnabled) {
```

Comment:
`createRecipeWebhook[0]`

you should not work like that. It's error-prone. 
If you know that there should be only one webhook wit a given name you should reflect that in the database (name unique)
So that when you fech the webhhok by it's name you should be sure that it's the one